### ISSUE TITLE
Pub Med Labs update to PubMed.js 

### ISSUE THREAD

**Issue by akinhwan** [zotero/translators#1800]
Hello Zotero translators!

A Zotero user has indicated that they are not getting the metadata properly ingested from Pub Med Labs. [Pub Med Labs](https://www.ncbi.nlm.nih.gov/labs/pubmed/) is the experimental site that will eventually replace PubMed. I am one of the developers on the project. This is my first time using Zotero personally, and would appreciate any an all input from the Zotero community.

Below are some of the changes I am planning to PR. but thought I'd open an issue to have any discussion/questions here. 

- update target regex so that PubMedTranslator.js fires on PubMedLabs (hereafter PML) urls

- current PubMed.js translator looks for meta tag with name="ncbi_uidlist". However on the abstract page it contains single article IDs, while on the docsum page - multiple IDs. We have the same data in a meta tag with a different name. I could either update the PML meta or make the translator support PML meta.

- translator.js has function getSearchResults that extracts from search results page array of tuples (article id, article title), and it is tied closely to PubMed HTML, and needs updating to support PML (in graceful manner based on data- attributes rather than CSS class names/HTML layout)

**Comment by akinhwan** [comment]
Attached is a document with screenshots outlining what the user's complaints were.
[Zotero.docx](https://github.com/zotero/translators/files/2696089/Zotero.docx)

**Comment by adam3smith** [comment]
Thanks for being in touch!
That all sounds great to me for a PR. 1 is a no-brainer. For 2 and 3, I'd be happy for you to just include alternate logic for PML into the existing PubMed translator. I'd probably make that logic contingent on the CSS/metatags rather than on the URL so that we don't have to worry about the timing of the PubMed to PML switch and have this working continuously.

The underlying eutils API is remaining unchanged, right?

**Comment by akinhwan** [comment]
@adam3smith So when I click on the Zotero Connector Chrome Extension icon, I see the following screenshot
![saving_zotero](https://user-images.githubusercontent.com/10124722/50251322-82333700-03b1-11e9-9db6-4e127f2fe442.PNG)
Which does not go away, and I don't ever see anything saved either on my Desktop Zotero Application, or in my zotero.org account. Is there some step I missed in connecting these?

I also downloaded the Scaffold Test framework, and was reading the (support/dev article)[https://www.zotero.org/support/dev/translators/scaffold] about it, which apparently may be outdated. But not sure how to run a testCase I wrote in PubMed.js. What are the suggested steps? 

In the "browser" tab of Zotero Scaffold, I've typed in the URL: `chrome://zotero/content/tools/testTranslators/testTranslators.html`
and clicked "Run" for PubMed, but I don't think that reflects my changes. Should I just paste my updated PubMed.js in the "Code" tab?
